src/search/pdbs/pattern_database.h

```diff
@@ -82,6 +92,10 @@ class PatternDatabase {
     */
     std::vector<int> distances;
 
+    std::vector<int> generating_op_ids;
+    std::vector<int> number_of_zero_cost_ops;
+    std::vector<std::vector<OperatorID>> wildcard_plan;
```

Comment:
Wouldn't it make sense to decouple PDB creation from PDB lookup first (as proposed in another issue)? Otherwise, all other pattern generators will need to store these extra members for each PDB even though they don't need them.

Reply:
More generally, having data that is only relevant some of the time is a bit of a code smell. Not something we need to address right now, but something to keep in mind for future refactoring.

Reply:
We discussed yesterday that ideally, one would do the other issue first, but at the same time, we wanted to not keep too many issues open waiting for other issues to be integrated first. The plan is to directly continue working on the other issue.

Reply:
Sounds good.